src/shared/components/case-editor/services/work-allocation.service.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { Response, ResponseOptions } from '@angular/http';
-import { Observable, throwError } from 'rxjs';
+import { Observable, of, throwError } from 'rxjs';
```

Comment:
I'm not seeing where you're using `of` in here. You're always using `Observable.of`, which makes the `of` import redundant.

Reply:
Agreed